### ISSUE TITLE
waybar 0.9.6 segfaults on startup

### ISSUE THREAD

**Issue by jpalus** [Alexays/Waybar#1071]
Just upgraded waybar to 0.9.6 on my ARM laptop and it segfaults immediately. Seems to be related to CPU frequency support added in #1037 which does not seem to cover systems which do not have frequencies in /proc/cpuinfo ie:
```
$ cat /proc/cpuinfo 
processor       : 0
BogoMIPS        : 48.00
Features        : fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0xd03
CPU revision    : 4
```
Note that it would be good  if waybar didn't even try to look for frequency if I don't use it in format string.

**Comment by Alexays** [comment]
Hi, can you test with this PR https://github.com/Alexays/Waybar/pull/1070
If it's good i'll release a new version with this fix

**Comment by jpalus** [comment]
Confirmed that it fixes startup issue.

**Comment by Alexays** [comment]
Closed as #1070 is merged
